src/main/kotlin/com/zepben/evolve/metrics/SourceMetadata.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.metrics
+
+import java.time.Instant
+
+/**
+ * Metadata for a data source used in ingestion.
+ *
+ * @property timestamp The time the source was exported from the source system.
+ * @property fileHash SHA-256 of the file. // TODO check if this is the correct type of SHA to use
```

Comment:
Are we using SHA-256 for source file hashes?

Reply:
yep